install/resources/mybb_theme.xml

```diff
@@ -822,6 +822,14 @@ blockquote cite span.highlight {
 	font-size: 13px;
 }
 
+.mycode_me {
```

Comment:
By creating a `mycode.css` stylesheet attached to `showthread.php`, `newthread.php`, `newreply.php` etc. it would mean themes without the stylesheet will inherit from the master style. So this change would be able to be included in a minor update...

Reference: http://community.mybb.com/thread-190334-post-1215259.html#pid1215259